Nodejs/Product/Nodejs/TypeScriptHelpers/TsConfigJson.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Diagnostics;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.NodejsTools.TypeScript
+{
+    [DebuggerDisplay("{FilePath}")]
+    internal sealed class TsConfigJson
+    {
+        public readonly bool NoEmit;
+
+        public readonly string OutFile;
+
+        public readonly string OutDir;
+
+        public readonly string RootDir;
+
+        public readonly string FilePath;
+
+        public TsConfigJson(string filePath, dynamic package)
```

Comment:
>dynamic [](start = 45, length = 7)

Why are we using `dynamic`?

Reply:
Looks like it could be `JObject`.

---
In reply to: [172939892](https://github.com/Microsoft/nodejstools/pull/1886#discussion_r172939892) [](ancestors = 172939892)